src/metaschema/oscal_mapping-common_metaschema.xml

```diff
@@ -0,0 +1,123 @@
+    <schema-name>OSCAL Mapping Model -- Common Models</schema-name>
+    <schema-version>1.0.0</schema-version>
+    <short-name>oscal-mapping-common</short-name>
+    <namespace>http://csrc.nist.gov/ns/oscal/1.0</namespace>
+    <json-base-uri>http://csrc.nist.gov/ns/oscal</json-base-uri>
+
+    <import href="oscal_metadata_metaschema.xml"/>
+
+    <define-assembly name="map">
+        <formal-name>Mapping Entry</formal-name>
+        <description>An individual entry that is part of a larger mapping.</description>
+        <define-flag name="uuid" as-type="uuid" required="yes">
+            <formal-name>Mapping Entry Identifier</formal-name>
+            <description>The unique identifier for the mapping entry.</description>
+        </define-flag>
+        <model>
+            <assembly ref="property" max-occurs="unbounded">
+                <group-as name="props" in-json="ARRAY"/>
+            </assembly>
+            <assembly ref="link" max-occurs="unbounded">
+                <group-as name="links" in-json="ARRAY"/>
+            </assembly>
+            <define-field name="relationship" as-type="token" min-occurs="1">
+                <formal-name>Mapping Entry Relationship</formal-name>
+                <description>The relationship type for the mapping entry.</description>
+                <define-flag name="ns" as-type="uri">
+                    <formal-name>Relationship Value Namespace</formal-name>
+                    <description>A namespace qualifying the relationship's value. This allows different organizations to associate distinct semantics for relationships with the same name.</description>
+                    <remarks>
+                        <p>An organization MUST use a URI that they have control over. e.g., a domain registered to the organization in a URI, a registered uniform resource names (URN) namespace.</p>
+                        <p>When a <code>ns</code> is not provided, its value should be assumed to be <code>http://csrc.nist.gov/ns/oscal</code> and the name should be a name defined by the associated OSCAL model.</p>
+                    </remarks>
+                </define-flag>
+                <constraint>
+                    <allowed-values target=".[has-oscal-namespace('http://csrc.nist.gov/ns/oscal')]">
+                        <enum value="equivalent-to">The source is equivalent in semantic meaning to the target. The words may differ, but both mapped elements have the same effective meaning.</enum>
+                        <enum value="equal-to">The source is the same as the target. Differences in capitalization, spelling, and grammar can be ignored, if these differences do not change the meaning.</enum>
+                        <enum value="subset-of">The source is a semantic subset of the target.</enum>
```

Comment:
Related to the note above (since some people I ran it by where analyzing based on *options of satisfying the _source_ vs the _tatget_ and not based on the set of requirements provided by the _source_ vs _target_) would it be more clear if the statement you read: "The set of requirements or information provided by the source is a semantic subset of the set of requirements or information provided by the target."
We might want to clarify that "this relationship is NOT considered bidirectional or automatically reversible to `superset-of`"

Reply:
Adjusting the text to be:

> The effective requirements of the source is a semantic subset of the effective requirements of the target. This relationship may be reversed as a `superset-of`, since `A subset-of B` also means that `B superset-of A`.

A subset-of B is reversible to B superset-of A. This is provable in basic set theory.

Reply:
Done in latest commit.